v2/schema/options.schema.yml

```diff
@@ -37,6 +37,7 @@ items:
           type: "string"
           enum:
             - "systemimagechannels"
+            - "lineage_oschannels"
```

Comment:
I'm a bit unsure about this line
@NeoTheThird what do you think?

Reply:
uh, realizing this entire section actually does not work ^^ lemme whip up a fix